pkg/chunk/chunk_store_test.go

```diff
@@ -35,32 +33,33 @@ func newTestChunkStore(t *testing.T, cfg StoreConfig) *Store {
 	return store
 }
 
-func createSampleStreamIteratorFrom(chunk Chunk) (local.SeriesIterator, error) {
+func createSampleStreamFrom(chunk Chunk) (*model.SampleStream, error) {
 	samples, err := chunk.Samples()
 	if err != nil {
 		return nil, err
 	}
-	return util.NewSampleStreamIterator(&model.SampleStream{
+	return &model.SampleStream{
 		Metric: chunk.Metric,
 		Values: samples,
-	}), nil
+	}, nil
 }
 
 // Allow sorting of local.SeriesIterator by fingerprint (for comparisation tests)
-type ByFingerprint []local.SeriesIterator
+type ByFingerprint model.Matrix
 
-func (s ByFingerprint) Len() int {
-	return len(s)
+func (bfp ByFingerprint) Len() int {
+	return len(bfp)
 }
-func (s ByFingerprint) Swap(i, j int) {
-	s[i], s[j] = s[j], s[i]
+func (bfp ByFingerprint) Swap(i, j int) {
+	bfp[i], bfp[j] = bfp[j], bfp[i]
 }
-func (s ByFingerprint) Less(i, j int) bool {
-	return s[i].Metric().Metric.Fingerprint() < s[j].Metric().Metric.Fingerprint()
+func (bfp ByFingerprint) Less(i, j int) bool {
+	return bfp[i].Metric.Fingerprint() < bfp[j].Metric.Fingerprint()
 }
 
-// TestChunkStore_Get tests iterators are returned correctly depending on the type of query
-func TestChunkStore_Get_concrete(t *testing.T) {
+// TODO(prom2): reintroduce tests that were part of TestChunkStore_Get_lazy
```

Comment:
Given that our next step is testing, it would be good to actually reintroduce these.

Reply:
Ack.